### PR TITLE
OADP-844: Adding to known issues - generated route can have incorrect…

### PR DESCRIPTION
**Pull request by anarnold97**
### Jira 
* [OADP-844](https://issues.redhat.com/browse/OADP-844)

### Previous PR

[Previous PR](https://github.com/openshift/openshift-docs/pull/65626) was approved by QE and Peer Review done by https://github.com/skrthomas 

### Version(s):
Enterprise 4.11 → branch/enterprise-4.11
Enterprise 4.12 → branch/enterprise-4.12
Enterprise 4.13 → branch/enterprise-4.13
Enterprise 4.14 → branch/enterprise-4.14
Enterprise 4.15 → branch/enterprise-4.15

#### Issue:
https://issues.redhat.com/browse/OADP-844

### Link to docs preview:
* [OADP 1.2.0 release notes](https://68867--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/oadp-release-notes#oadp-release-notes-1-2-0_oadp-release-notes)

### QE review:
* [X ] QE has approved this change.
* QE approved by Sachin in previous [PR#65626 - QE LGTM ](https://github.com/openshift/openshift-docs/pull/65626#pullrequestreview-1661213263)
* Peer review approved in previous PR = [LGTM](https://github.com/openshift/openshift-docs/pull/65626#pullrequestreview-1664832364)

### CODE REVIEW

File: modules/oadp-release-notes-1-2-0.adoc [hunk 1]
```diff
@@ -38,12 +39,20 @@ The following issues have been highlighted as known issues in the release of OAD
...
+The HTTP/2 protocol is susceptible to a denial of service attack because request cancellation can reset multiple streams quickly. The server has to set up and tear down the streams while not hitting any server-side limit for the maximum number of active streams per connection. This results in a denial of service due to server resource consumption.
 
 It is advised to upgrade to OADP 1.2.3, which resolves this issue.
 
 For more information, see link:https://access.redhat.com/security/cve/cve-2023-39325[CVE-2023-39325 (Rapid Reset Attack)].
 
+.An incorrect hostname can be created when changing a hostname in a generated route.
+
+By default, OpenShift cluster makes sure that the openshift.io/host.generated: `true` annotation is turned on and fills in the field for both the routes that are generated and those that are not generated.
```
Comment (jeana-redhat) [inline modules/oadp-release-notes-1-2-0.adoc:50]:
```suggestion
By default, the {product-title} cluster makes sure that the` openshift.io/host.generated: true` annotation is turned on and fills in the field for both the routes that are generated and those that are not generated.
```

File: modules/oadp-release-notes-1-2-0.adoc [hunk 2]
```diff
@@ -38,12 +39,20 @@ The following issues have been highlighted as known issues in the release of OAD
...
+
+You cannot modify the value for the `.spec.host` field based on the base domain name of your cluster in the generated and non-generated routes.
+
+If you modify the value for the `.spec.host` field, it is not possible to restore the default value that was generated by the OpenShift cluster. After you restore your OpenShift cluster, the operator resets the value for the field.
```
Comment (jeana-redhat) [inline modules/oadp-release-notes-1-2-0.adoc:54]:
```suggestion
If you modify the value for the `.spec.host` field, it is not possible to restore the default value that was generated by the {product-title} cluster. After you restore your {product-title} cluster, the Operator resets the value for the field.
```

**Review by jeana-redhat — COMMENTED** [review]
Just a couple things to fix here before merge

/remove-label merge-review-in-progress
/remove-label merge-review-needed

**Comment by anarnold97** [comment]
@jeana-redhat - Thanks as always

*NOTE TO SELF* remember to use attributes, ye daft eejit

**Comment by anarnold97** [comment]
@jeana-redhat - i am sure it was deliberate, to make sure we were paying attention :)

**Comment by anarnold97** [comment]
@jeana-redhat - WOOT-WOOT! thanks very much

**Comment by jeana-redhat** [comment]
@anarnold97 np! looks like 4.11 needs manual intervention but I will merge the rest when they build

**Comment by anarnold97** [comment]
@jeana-redhat - thanks. 
i will sort 4.11 out first thing tomorrow morning

**Pull request merged**